templates/default/capture-agent-manager-supervisor-conf.erb

```diff
@@ -0,0 +1,14 @@
+[program:<%= @capture_agent_manager_name %>]
+command = /home/<%= @capture_agent_manager_usr_name %>/sites/<%= @capture_agent_manager_name %>/gunicorn_start.sh
+directory = /home/<%= @capture_agent_manager_usr_name %>/sites/<%= @capture_agent_manager_name %>
+user = <%= @capture_agent_manager_usr_name %>
+redirect_stderr = True
+
+stdout_logfile = /home/<%= @capture_agent_manager_usr_name %>/logs/<%= @capture_agent_manager_name %>_gunicorn_stdout.log
+stderr_logfile = /home/<%= @capture_agent_manager_usr_name %>/logs/<%= @capture_agent_manager_name %>_gunicorn_stderr.log
```

Comment:
I don't think you need this since you've got `redirect_stderr = True`. Everything is going to go to `stdout_logfile`.